UIImageColorsPlayground.playground/Sources/UIImageColors.swift

```diff
@@ -25,21 +25,33 @@ class PCCountedColor {
     }
 }
 
-extension UIColor {
+extension UIColor
+{
+    var components:[CGFloat]
```

Comment:
If memory serves I added that when they broke the `cgColor`  one as a workaround, I thought I removed it but apparently I was wrong. Done.